kalite/distributed/static/js/distributed/base/handlebars.js

```diff
@@ -113,4 +113,8 @@ Handlebars.registerHelper("ifObject", function(candidate, options){
     }
 });
 
+Handlebars.registerHelper("current_language_is", function(lc, options) {
+    return (window.sessionModel.get("CURRENT_LANGUAGE") == lc)?options.fn(this):options.inverse(this);
```

Comment:
I just followed the formatting used in the handlebars.js file, like line number 9.

But, okay sir. I will follow your suggestion.